src/theme/edgeConfig.ts

```diff
@@ -17,6 +17,7 @@ export const edgeConfig: AppConfig = {
     { pluginId: 'bitcoincash', tokenId: null },
     { pluginId: 'dash', tokenId: null }
   ],
+  expiredPromoUrl: 'https://edge.app/promo-expired', // TBD
```

Comment:
Be sure to coordinate on this.

Reply:
Per the most recent discussion, we now want all relevant promo cards that are expired to end up in the notification center, in addition to dismissed active promotions.